src/runtimeContext.ts

```diff
@@ -27,6 +27,12 @@ export function getVueConstructor(): VueConstructor {
 }
 
 export function setVueConstructor(Vue: VueConstructor) {
+  if (__DEV__ && vueConstructor) {
+    // don't warn in jest
+    if ('jest' in global) {
```

Comment:
Is this too special? I think it might need to be handle on the user side.

Reply:
I feel it would be annoying you getting the warning when testing, because you will always have the warning if you use `localVue`

Reply:
Hmm, then how about `ava` `mocha`, etc? I am not very sure about this.

Reply:
me neither, we have two choices, warn every time or not warn at all

Reply:
I would stay just warn every time. And let users who wrote the tests to handle it. (didn't try it, but I guess ava runs in isolated env parallelly, which might not have this issue). Anyway, I feel it's better to warn as ppl might need to be sure what they are doing.